src/main/java/organice/logic/commands/ExactFindCommand.java

```diff
@@ -0,0 +1,44 @@
+package organice.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import organice.commons.core.Messages;
+import organice.model.Model;
+import organice.model.person.PersonContainsPrefixesPredicate;
+
+/**
+ * Finds and lists all persons in address book whose prefixes match any of the argument prefix-keyword pairs.
+ * Keyword matching is case insensitive.
+ */
+public class ExactFindCommand extends Command {
+
+    public static final String COMMAND_WORD = "exactFind";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose prefixes match any of "
+            + "the specified prefix-keywords pairs (case-insensitive) and displays them as a list with index numbers.\n"
+            + "List of Prefixes: n/, ic/, p/, a/, t/, pr/, b/, d/, tt/, exp/, o/"
+            + "Parameters: PREFIX/KEYWORD [MORE_PREFIX-KEYWORD_PAIRS]...\n"
+            + "Example: " + COMMAND_WORD + " n/alice t/doctor";
+
+    private final PersonContainsPrefixesPredicate predicate;
+
+    public ExactFindCommand(PersonContainsPrefixesPredicate predicate) {
+        requireNonNull(predicate);
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredPersonList(predicate);
```

Comment:
You may need to add `model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS)` to reset the FilteredList before applying your Predicate.

Reply:
updating the predicate does not change the total number of items inside the filtered persons. FilteredList already takes care of the situation you are thinking of